integrations/airflow/tests/integration/integration.py

```diff
@@ -29,7 +25,7 @@
 
 log = logging.getLogger(__name__)
 
-NAMESPACE_NAME = 'food_delivery'
+JOB_NAMESPACE_NAME = 'food_delivery'
```

Comment:
_Minor:_ You can still keep `NAMESPACE_NAME` as it's apparent in the tests that the source name is used for the namespace in some cases. Also, reduces the diff in the PR.

Reply:
Done.